setup.py

```diff
@@ -8,7 +8,7 @@
 # NOTE: These are tested in `continuous_integration/travis/test_imports.sh` If
 # you modify these, make sure to change the corresponding line there.
 extras_require = {
-  'array': ['numpy >= 1.11.0', 'toolz >= 0.7.3'],
+  'array': ['numpy >= 1.11.0', 'toolz >= 0.7.3', 'crick >= 0.0.3'],
```

Comment:
crick should be an optional dependency for dask array, and probably an optional dependency for dask dataframe (for now).

Reply:
Removed.